src/emr-serverless/activation.ts

```diff
@@ -0,0 +1,38 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import * as vscode from 'vscode'
+import { EmrServerlessJobNode } from './explorer/emrServerlessJobNode'
+import { getDashboard } from './commands/getDashboard'
+import { Commands } from '../shared/vscode/commands2'
+import { EmrServerlessApplicationNode } from './explorer/emrServerlessApplicationNode'
+import { copyTextCommand } from '../awsexplorer/commands/copyText'
+import { startApplication } from './commands/startApplication'
+import { stopApplication } from './commands/stopApplication'
+
+/**
+ * Activates EMR Serverless components.
+ */
+export async function activate(extensionContext: vscode.ExtensionContext): Promise<void> {
+    extensionContext.subscriptions.push(
+        Commands.register('aws.emrserverless.getDashboard', async (node: EmrServerlessJobNode) => {
+            await getDashboard(node)
+        }),
+
+        Commands.register(
+            'aws.emrserverless.copyId',
+            async (node: EmrServerlessApplicationNode | EmrServerlessJobNode) => {
+                await copyTextCommand(node, 'id')
+            }
+        ),
+
+        Commands.register('aws.emrserverless.startApplication', async (node: EmrServerlessApplicationNode) => {
```

Comment:
could this be one parameterized command instead of 2 commands?

Reply:
Could probably do that by checking node state maybe? Is there a way to update the command text accordingly? e.g. "Start Application" on a stopped node and vice-versa?